main.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"final13/pkg/api"
+	"final13/pkg/db"
+	"final13/tests"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/joho/godotenv"
+)
+
+func main() {
+	// Initialization of port and DB
+	port := os.Getenv("TODO_PORT")
+	if port == "" {
+		port = strconv.Itoa(tests.Port)
+	}
+
+	dbFile := os.Getenv("TODO_DBFILE")
+	if dbFile == "" {
+		dbFile = tests.DBFile
```

Comment:
Так себе идея, по умолчанию БД в проекте лежит, а тесты ходят из своей папки на уровень выше в проекта
```suggestion
		dbFile = "scheduler.db"
```